### PR TITLE
Issue #255: Payment test data

### PR DESCRIPTION
**Pull request by jucron**
## 🚀 Description

Hello friend! I wanted to contribute to this project, so I did the tests for Payment Services. 

## 📄 Motivation and Context

Motivation: 
- Currently default test data for payments is being missed. This hinders the development flow. 
Context: 
- Added a file called PaymentSDJpaServiceTest.java based on existing PaymentSDJpaService.java with all tests. Changed - TestUtils.java file at the end with some helper code.
- Note: wanted to implement a efficient UnitTest avoiding an IT, so I had to capture the parameters in the repository method in order for them to be verified.
Thank you so much for the opportunity to collaborate!
Issue link (https://github.com/jmprathab/MyHome/issues/255)

## 🧪 How Has This Been Tested?

Basically had to log some parameters after writing the code, and go by test-driven-development. Assertions got me secured that everything was fine.

## 📷 Screenshots (if appropriate)

image

## 📦 Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

- [ ] My code follows the code style of this project(Do your best to follow code styles. If none apply just skip this).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by jucron** [comment]
I'm sending a new PR with the correct branch's name.
